app/code/Magento/Catalog/Model/ImageUploader.php

```diff
@@ -174,6 +172,14 @@ public function getAllowedExtensions()
         return $this->allowedExtensions;
     }
 
+    /**
+     * @return string[]
+     */
+    public function getAllowedMimeTypes()
```

Comment:
Why need such getter?

Reply:
ok, we do not need it. i have followed the convention by $allowedExtensions